app/views/alaveteli_pro/plans/show.html.erb

```diff
@@ -12,7 +12,7 @@
           data-key="<%= AlaveteliConfiguration.stripe_publishable_key %>"
           data-name="<%= AlaveteliConfiguration.pro_site_name %>"
           data-description="A monthly subscription"
-          data-amount="<%= @plan.amount %>"
+          data-amount="<%= @display_price %>"
```

Comment:
it tried to charge me 10p instead of £10 until I added ` * 100`

Reply:
soz that's from the WIP VAT commit

Reply:
just noting so it doesn't get lost!